app/models/order.rb

```diff
@@ -336,4 +353,11 @@ def order
   def offer
     nil
   end
+
+  def last_end_of_work_day
+    now = Time.now.in_time_zone
+    last_6pm = now.change(hour: 18, min: 0, sec: 0)
```

Comment:
I think you can do Time.now.yesterday and use that as a base. But this works too.